docker-compose-migration.yml

```diff
@@ -1,17 +0,0 @@
-version: '2'
```

Comment:
What the point to remove this docker file?

Reply:
because my solution don't need them